src/test/suite/runTestSubset.ts

```diff
@@ -4,7 +4,7 @@
  * configuration.
  * See https://mochajs.org/#-grep-regexp-g-regexp for supported syntax
  */
-export const TEST_SUBSET_GREP_STRING = "implicit";
+export const TEST_SUBSET_GREP_STRING = "Grapheme";
```

Comment:
This bastard gets updated all the time apparently :D

Reply:
haha yeah I wonder if it's worth coming up with some alternative.  Doesn't seem like too big of a deal tho i guess 🤷‍♂️

Reply:
No it's fine

Reply:
Would probably be nicer to make it purely a CLI thing, but we can kick the can down the road a bit.

Reply:
How would we make it a CLI thing?  This variable only has an impact when you run a particular VSCode launch config

Reply:
Launch configs can accept input, see https://code.visualstudio.com/docs/editor/variables-reference#_input-variables

Reply:
that's an interesting idea.  Makes it a bit harder to run the same set of tests over and over again, but does avoid this git fight